DigitalLearningSolutions.Data/Models/User/AccountDetailsData.cs

```diff
@@ -3,30 +3,18 @@
     public class AccountDetailsData
     {
         public AccountDetailsData(

```

Comment:
If we do make this class abstract, this constructor should probably be protected.